spigot-tests/tests/utils/test-utils.ts

```diff
@@ -494,7 +497,7 @@ export async function attachStream (stream: MediaStream, reattach?: boolean, des
   logger.debug('starting auto play of element', stream);
 
   // sometimes this doesn't play for whatever reason
-  el.play()
+  // await el.play()
```

Comment:
Is it intentional for this to be commented out?

Reply:
it was intentional. I did this when I was running into issues and it didn't seem to make a difference either way and *shouldn't* be needed.